server/appengine/src/main/java/who/RefreshCaseStatsServlet.java

```diff
@@ -109,5 +151,38 @@ private String getDashboardContents() throws IOException {
         .attribution("WHO");
 
     StoredCaseStats.save(global.build());
+
+    for (Map.Entry<String, HashMap<Long, StoredCaseStats.StoredStatSnapshot>> outerEntry : allCountrySnapshots.entrySet()) {
+      
+      String iso2code = outerEntry.getKey();
+      HashMap<Long, StoredCaseStats.StoredStatSnapshot> countrySnapshots = outerEntry.getValue();
+      
+      long countryTotalCases = 0L;
+      long countryTotalDeaths = 0L;
+
+      for (Map.Entry<Long, StoredCaseStats.StoredStatSnapshot> innerEntry : countrySnapshots.entrySet()) {
```

Comment:
is this aggregation correct? 

it is based on the assumption that either the snapshots are per day or incremental updates daily.

Can someone please confirm?